src/base/preferences.cpp

```diff
@@ -1429,6 +1429,15 @@ void Preferences::setSearchTabHeaderState(const QByteArray &state)
     setValue("SearchTab/qt5/HeaderState", state);
 }
 
+bool Preferences::getRegexAsFilteringPatternForSearchJob() const
```

Comment:
I think you can omit these additional functions (in both cases). Why not use SettingsStorage directly instead?

Reply:
ok.